templates/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     metadata:
       annotations:
         {{- if .Values.pod.annotations }}
-        {{ toYaml .Values.pod.annotations | indent 8 }}
+        {{ toYaml .Values.pod.annotations | nindent 8 }}
```

Comment:
@take-five Can you change it to:
```
...
      annotations:
        {{- if .Values.pod.annotations }}
        {{- toYaml .Values.pod.annotations | nindent 8 }}
        {{- end }}
```

This would get rid of any extra empty lines.

Reply:
@take-five are you able to action on this request? Thank you!

Reply:
@p0pr0ck5 Yes, I updated my PR. Sorry, somehow lost the first suggestion by @m-barczyk in my inbox :-)

Reply:
Thanks! We will pick up reviewing this this week. Thanks for your effort on this!